pulp_rpm/app/migrations/0054_remove_rpmpublication_gpgcheck_and_more.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 4.2.4 on 2023-09-08 14:45
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("rpm", "0053_rpmdistribution_generate_repo_config"),
+    ]
```

Comment:
We should move the settings over first, if they're present.  So add the fields first, then move, then remove the old ones.

Reply:
Also if we're trying to do "zero downtime" then it's more complicated.  I don't think we've really done that thus far and I don't think services have ever used these settings to begin with though.

Reply:
I was thinking about this, but came to conclusion to just remove these settings given that we have disabled the .repo file generation anyway.

Reply:
That sounds OK.